packages/ubuntu_desktop_installer/test/who_are_you/who_are_you_page_test.dart

```diff
@@ -93,9 +95,12 @@ void main() {
     final textField = find.widgetWithText(TextField, 'passwd');
     expect(textField, findsNWidgets(2));
     await tester.enterText(textField.first, 'ubuntu');
-    verify(model.password = 'ubuntu').called(1);
+    verify(model.setPassword(ValidatedString('ubuntu', isValid: true)))
+        .called(1);
     await tester.enterText(textField.last, 'ubuntu');
-    verify(model.confirmedPassword = 'ubuntu').called(1);
+    verify(model
+            .setConfirmedPassword(ValidatedString('ubuntu', isValid: false)))
```

Comment:
Same question here, why is the expectation that the confirmed password is invalid?